Apps/WebClient/src/ClientApp/app/views/timeline.vue

```diff
@@ -388,20 +391,26 @@ export default class TimelineComponent extends Vue {
   }
 
   private get getNumberOfEntriesPerPage(): number {
-    if (this.windowWidth < 576) { // xs

```

Comment:
these numbers need to be constants. I'am guessing this is not part of the issue fix. Lets create a ticket to address this.

Reply:
Agreed AB#8168 created.